nntrainer/src/pooling2d_layer.cpp

```diff
@@ -156,8 +204,13 @@ Tensor Pooling2DLayer::pooling2d(Tensor in, int &status) {
           for (unsigned int pi = 0; pi < p_height; ++pi) {
             for (unsigned int pj = 0; pj < p_width; ++pj) {
               float val = in.getValue(0, i, j + pi, k + pj);
```

Comment:
Could you double check this as well?
```suggestion
              float val = in.getValue(batch, i, j + pi, k + pj);
```

Reply:
I checked that `in` has always batch size of 1

Reply:
batch is right.